doc/user_guide/io.rst

```diff
@@ -440,3 +442,65 @@ format stores all the captured data for each timestamp, together with a small
 header in a plain-text format. The reader extracts the measured temperature
 along the time axis, as well as the date and calibration constants stored in
 the header.
+
+
+.. _bcf-format:
+
+Bruker composite file
+----------------
+
+HyperSpy can read "hypermaps" saved with Bruker's Esprit v1.x or v2.x in bcf
+hybrid (virtual file system/container with xml and binary data, optionaly compressed) format.
+Most of relevant data extraction from bcf is implemented. Both: high
+resolution 16-bit SEM imagery and hyperspectral EDX data can be retrieved simultaniously.
+
+Note that Bruker Esprit uses the similar format for EBSD data, but even if technology
+is very similar, curently EBSD containing bcf is not supported by HyperSpy.
```

Comment:
I think something along the lines of below reads better?

"Most bcf import functionality is implemented. Both high-resolution 16-bit SEM images and hyperspectral EDX data can be retrieved simultaneously.

Note that Bruker Esprit uses a similar format for EBSD data, but it is not currently supported by HyperSpy."